ctf_framework/pipelines.py

```diff
@@ -25,21 +23,15 @@ def login_user(request, api_data):
 
         # Create a guaranteed unique and static django username from slack Team ID and User ID
         user, created = User.objects.get_or_create(
-            username="{}:{}".format(api_data['team_id'], api_data['user_id'])
+            username="SLACK:{}:{}".format(api_data['team_id'], api_data['user_id'])
```

Comment:
Is the "SLACK" prefix necessary? What did you have in mind for that?

Reply:
Purely for identifying if we ever need the info. It would separate Slack for Google or any other login. This username is never used anywhere other than the core django users table.

Reply:
In the meantime, we should remove it as its not useful at the moment.